### DISCUSSION TITLE
No ID TOKEN and REFRESH TOKEN received

### DISCUSSION THREAD

**Discussion by andreamarinaro** [OpenIDC/mod_auth_openidc#519] [Q&A]
Hi all,
we are using cognito with openidc module but we are not able to execute the call to token endpoint of the Cognito (POST /oauth2/token) after the auth call(GET /oauth2/authorize).
Due to this we are not able to generate ID TOKEN and REFRESH TOKEN, even if the authentication part via Google SSO is correctly working.

This is our conf:

    OIDCRedirectURI https://xxx-xxx.xxxxxx.xxx/redirect_uri
    OIDCClientID xxxxxxxxxxxxxxxxxxxxxxxxx
    OIDCClientSecret xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
    OIDCProviderMetadataURL xxxxxxxxxxxxxxxxxxxxxxxxxx
    OIDCCryptoPassphrase xxxxxx
    OIDCSessionMaxDuration 120
    OIDCCookiePath /
    OIDCCacheType memcache
    OIDCMemCacheServers memcached-cognito-xxxxxxxxx.xxxx.amazonaws.com:11211

    <Location />
     AuthType openid-connect
     Require valid-user
    </Location>

Thanks a lot

**Comment by zandbelt** [comment]
you may want to inspect the Apache server debug logs to see what is happening
